cli.mjs

```diff
@@ -62,11 +67,19 @@ loadSSL(opts)
     process.exit(1);
   });
 
+if(sshkey) {
+  console.warn('!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n' +
```

Comment:
Bit nitpicky can we use a multi line string template

Reply:
It quite difficult to make the indentation right with multiline strings.
I rewrote it as:
```
const sshkeyWarning =
`!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
! Password-less auth enabled using private key from \'%s\'.
! This is dangerous, anything that reaches the wetty server
! will be able to run remote operations without authentication.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!`;
if(sshkey) {
  console.warn(sshkeyWarning);
}
```

Reply:
I could move the const within the if block, but as to not add spaces the lines [2,4] would need to remain without indent, so it would look a bit more weird.